AnswerDotAI/llms-txt
# look at local files first

Problem: When you build the context right now, it pulls from the URL.  However, what if the URL correpsonds to the current nbdev repo?  We should be smart enough to pull from the local generated docs rather than the URL so we don't have to run things twice!  

If we want to automate the `llms-ctx*.txt` creation when we build the docs, I think we need a way to do this?

How to test:

1. Install this branch
2. Go to a nbdev repo, make a local change to a notebook. 
3. Run `nbdev_docs`
4. Run `llms_txt2ctx` tool, you will see your local changes being reflected appropriately
